oauth2_provider/constants.py

```diff
@@ -41,6 +41,10 @@
 SCOPE_NAME_DICT = dict([(name, value) for (value, name) in SCOPES])
 SCOPE_VALUE_DICT = dict([(value, name) for (value, name) in SCOPES])
 
+# since this package explicitly supports the 'id_token' response type, override
+# the base package's default.
+DEFAULT_RESPONSE_TYPE_CHOICES = ('code', 'token', 'id_token')
+RESPONSE_TYPE_CHOICES = getattr(settings, 'OAUTH_RESPONSE_TYPE_CHOICES', DEFAULT_RESPONSE_TYPE_CHOICES)
```

Comment:
Where is the base package's default being "overridden"?
Won't it not be updated if `'OAUTH_RESPONSE_TYPE_CHOICES'` is already set?

I would have expected this code to simply add `id_token` to the existing `provider.constants.RESPONSE_TYPE_CHOICES`.